### PR TITLE
Vfov padding

### PR DESCRIPTION
**Pull request by nickovaras**
Compiles and runs as expected with 3D lidar.

### CODE REVIEW

File: src/spatio_temporal_voxel_layer.cpp [hunk 1]
```diff
@@ -200,11 +203,11 @@ void SpatioTemporalVoxelLayer::onInitialize(void)
...
         boost::shared_ptr <buffer::MeasurementBuffer>
-        (new buffer::MeasurementBuffer(topic, observation_keep_time,     \
-        expected_update_rate, min_obstacle_height, max_obstacle_height,  \
-        obstacle_range, *tf_, _global_frame,                             \
-        sensor_frame, transform_tolerance, min_z, max_z, vFOV,           \
-        hFOV, decay_acceleration, marking, clearing, _voxel_size,        \
+        (new buffer::MeasurementBuffer(topic, observation_keep_time,      \
+        expected_update_rate, min_obstacle_height, max_obstacle_height,   \
+        obstacle_range, *tf_, _global_frame,                              \
+        sensor_frame, transform_tolerance, min_z, max_z, vFOV,vFOVPadding,\
```
Comment (SteveMacenski) [inline src/spatio_temporal_voxel_layer.cpp]: Missing a space here, place rearrange to the next line

File: include/spatio_temporal_voxel_layer/frustum_models/three_dimensional_lidar_frustum.hpp [hunk 2]
```diff
@@ -52,8 +52,8 @@ namespace geometry
 class ThreeDimensionalLidarFrustum : public Frustum
 {
 public:
-  ThreeDimensionalLidarFrustum(const double& vFOV, const double& hFOV,
-          const double& min_dist, const double& max_dist);
+  ThreeDimensionalLidarFrustum(const double& vFOV,const double& vFOVPadding,
```
Comment (SteveMacenski) [inline include/spatio_temporal_voxel_layer/frustum_models/three_dimensional_lidar_frustum.hpp]: Looks like you deleted a space between `,` and `const`

**Comment by SteveMacenski** [comment]
Looks good, let me throw this on a robot tomorrow and verify it doesnt break anything on the depth sensor front

**Comment by nickovaras** [comment]
Awesome.

**Comment by nickvaras** [comment]
Any luck with this?

**Comment by SteveMacenski** [comment]
Doing right now, sorry for the delay

**Comment by nickvaras** [comment]
Thanks!

**Comment by SteveMacenski** [comment]
Looks good to me

**Comment by SteveMacenski** [comment]
Side note: if you had a little gif of your robot with the VLP-16 in rviz working, I'd more than welcome it in the readme as a demonstration

**Comment by nickovaras** [comment]
Here's a preview. Not the nicest gifs in the world, but they kind of convey the idea...

[https://github.com/nickovaras/spatio_temporal_voxel_layer/blob/vfov_padding/README.md](https://github.com/nickovaras/spatio_temporal_voxel_layer/blob/vfov_padding/README.md)

**Comment by SteveMacenski** [comment]
That's a really interesting thing that's moving in the second GIF. I'd be scared if something that size was honing towards my robots haha

**Comment by nickvaras** [comment]
Haha, It does look funny now that you mention it. It was a few 8'x4' pieces of material being carried on it's side by a couple people. Looks massive on the scan!

**Pull request merged**